src/teleop/gui/src/components/PDBFuse.vue

```diff
@@ -0,0 +1,144 @@
+<template>
+    <div class="wrap">
+      <div>
+        <h3>Power Distribution Board</h3>
+      </div>
+      <div class="table">
+        <table class="tableFormat" style="undefined;table-layout: fixed">
+          <colgroup>
+            <col style="width: 80px">
```

Comment:
Oh idk much about Vue but I feel like there's probably a way to use a for loop here to replace the hard coding of 0 1 and 2

Reply:
Yep maybe Joe has a different suggestion but should be something like v-for: https://www.geeksforgeeks.org/vue-js-list-rendering-v-for-on-a-template/

Reply:
I agree a v-for is the best way to clean this up